packages/system/etcd-operator-crds/Makefile

```
@@ -0,0 +1,13 @@
+export NAME=etcd-operator-crds
+export NAMESPACE=cozy-etcd-operator
+
+include ../../../hack/package.mk
+
+# Pinned ref of github.com/cozystack/etcd-operator to vendor CRDs from.
+ETCD_OPERATOR_REF ?= v0.5.0
+
+update:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)